app.js

```diff
@@ -5,12 +5,17 @@ const { Keccak } = require('sha3')
 const EC = require('elliptic').ec
 const http = require('http')
 
-let banned = require('./lib/utils/config').banned
-let reserved = require('./lib/utils/config').reserved
-let trusted = require('./lib/utils/config').trusted
-
-if (process.env.TRUSTED_NODE) {
-  trusted.push(process.env.TRUSTED_NODE)
+let trusted = []
+if (process.env.TRUSTED_ADDRESSES) {
```

Comment:
are we not reading the list from the config anymore?

probably we should append the env list with the config list